### PR TITLE
Auth Event Example

### PR DESCRIPTION
**Pull request by kebringer-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [X] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [X] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [X] A self-review of the code has been done.

## What is the ticket or issue number?
As part of release testing extended the policy example scripts to include Enabling and Disabling Auth Event Collection.

## Pull Request Description
New policy_data_collection example script.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## How Has This Been Tested?
Verified in the Carbon Black Console.

### CODE REVIEW

File: examples/platform/policy_data_collection.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+The API specification for underlying APIs is available on the Developer Network:
+https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/policy-service/#rule-config---data-collection
+"""
+
+import sys
+from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import Policy
+
+
+def list_policy_summaries(cb, args):
```
Comment (emitreva-cb) [inline examples/platform/policy_data_collection.py:37]: I am not seeing args being used in this function.

File: examples/platform/policy_data_collection.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            dcrc.save()
+            auth_event_rule_config = dcrc
+
+    if auth_event_rule_config is not None:
+        print(auth_event_rule_config)
+    else:
+        print("Auth Event Rule Config Element Not Found")
+
+
+def disable_auth_event_rule(cb):
```
Comment (emitreva-cb) [inline examples/platform/policy_data_collection.py:82]: I wonder whether the enable / disable functions can be merged and just one function to be used as they share most of the code, but even that seems ok.

Comment (kebringer-cb) [reply examples/platform/policy_data_collection.py]: agree.  merged.

File: examples/platform/policy_data_collection.py (hunk 1 above)
Comment (kebringer-cb) [reply examples/platform/policy_data_collection.py]: removed.

**Review by abowersox-cb — APPROVED** [review]

**Comment by kebringer-cb** [comment]
Approval from Ema via slack.

**Pull request merged**